packages/twenty-server/src/engine/api/graphql/workspace-query-builder/factories/args-string.factory.ts

```diff
@@ -55,22 +60,39 @@ export class ArgsStringFactory {
     return argsString;
   }
 
-  private buildStringifiedObject(
-    key: string,
-    obj: Record<string, any>,
+  private buildStringifiedObjectFromArray(
```

Comment:
`buildStringifiedOrderBy`? Because of the position logic, I feel this function is not a generic function such `buildStringifiedObjectFromArray`

Reply:
Okay!